pkgs/development/python-modules/smbus-cffi/default.nix

```diff
@@ -26,16 +26,18 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ cffi ];
 
+  checkInputs = [ pytestCheckHook pyserial ];
+
   installCheckPhase = ''
     # we want to import the installed module that also contains the compiled library
-    rm -rf smbus
+    rm -r smbu
```

Comment:
Mistake?

Reply:
No, ``rm -f`` should be avoided because it does not fail ever.

Reply:
No, I mean `smbu` -> `smbus`